docs: Grammar overhaul (#20636)
nuxt/nuxt: docs/1.getting-started/3.views.md

```

::alert
If you are familiar with Vue, you might wonder where `main.js` is (the file that creates a Vue app). Nuxt does this behind the scene.
::

## Components

![Components are reusable pieces of UI](/assets/docs/getting-started/views/components.svg)

> Most components are reusable pieces of the UI, like buttons and menus. In Nuxt, you can create these components in the `components/` directory. They will automatically be available across your application without explicitly importing them.

Reviewer: Consistency (other places are using _user interface_ rather than _UI_)
```suggestion
Most components are reusable pieces of the user interface, like buttons and menus. In Nuxt, you can create these components in the `components/` directory. They will automatically be available across your application (no need to import them explicitly).
```

Author: Implemented!

![Layouts are wrapper around pages](/assets/docs/getting-started/views/layouts.svg)

> Layouts are wrappers around pages that contain a shared User Interface for several pages, such as a header and footer display. Layouts are Vue files using `<slot />` components to display the **page** content. The `layouts/default.vue` file will be used by default. Custom layouts can be set as part of your page metadata.

Reviewer: ```suggestion
Layouts are wrappers around pages that contain a shared user interface for several pages, such as a header and footer. Layouts are Vue files using `<slot />` components to display the **page** content. The `layouts/default.vue` file will be used by default. Custom layouts can be set as part of your page metadata.
```
